APIs/RegistrationAPI.raml

```diff
@@ -34,6 +34,8 @@ documentation:
 
       The DNS-SD advertisement MUST be accompanied by a TXT record of name 'api\_ver'. The value of this TXT record is a comma separated list of API versions supported by the server. For example: 'v1.0,v1.1,v2.0'. There should be no whitespace between commas, and versions should be listed in ascending order.
 
+      When both defined service types are advertised, the following practice MUST be observed. All supported API versions MUST be listed against the service type \_nmos-register.\_tcp. All supported API versions up to and including (but not beyond) v1.2 MUST be listed against the service type \_nmos-registration.\_tcp.
```

Comment:
Thinking about it, I see no need to limit the versions advertised via _nmos-registration._tcp, so for simplicity (the values for both record names can be identical) I'd prefer to delete "(but not beyond)".